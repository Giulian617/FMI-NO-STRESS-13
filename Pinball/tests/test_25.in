100 78551965951241
12 79 3
96 79 3
90 96 10
72 96 8
6 90 8
14 90 2
11 12 3
83 6 6
66 90 7
5 90 9
76 90 10
2 83 8
19 2 4
45 2 3
95 83 8
43 72 8
21 12 1
50 11 1
37 76 1
73 72 4
8 96 8
25 66 3
4 12 8
31 11 9
86 50 2
65 95 9
23 79 2
74 90 3
46 25 7
24 66 9
82 43 9
55 46 10
69 19 7
27 11 7
99 83 10
62 19 4
1 69 3
67 74 4
59 65 9
28 45 7
33 6 9
49 25 2
58 4 2
48 11 3
85 55 5
78 37 1
81 90 3
88 69 8
61 62 5
75 67 4
87 67 4
70 58 2
53 67 7
36 74 5
44 14 8
29 88 2
64 24 4
56 81 10
60 25 1
92 81 2
7 8 4
91 11 2
3 2 5
77 50 8
39 37 3
22 53 5
68 4 4
32 86 9
30 44 1
93 24 9
40 56 5
47 55 8
89 30 2
84 88 10
63 12 8
71 59 4
98 61 7
54 64 6
38 82 6
26 55 6
94 56 4
80 62 10
97 82 10
42 40 6
57 6 3
18 63 2
51 33 6
16 68 9
15 27 2
13 53 10
41 59 1
100 99 10
20 25 2
9 38 5
34 7 6
52 86 3
35 19 7
17 91 7
10 95 4
