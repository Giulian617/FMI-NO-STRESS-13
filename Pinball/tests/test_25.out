670003365
323411618
395058945
576382247
285017273
242794111
941658055
379079532
638932383
11096931
514292809
288094136
596361455
3946461
860053628
664226975
677580491
541631237
982998973
745816677
154963517
666639371
591663234
1914185
563375492
399831772
183681394
514292809
569073470
925862248
3400096
335550685
53538572
972433403
806397942
590987791
505138907
493918450
320192081
6523750
687257890
144780729
505138907
955505837
570666500
598851917
108093472
191488597
538580506
701819343
